images/tensorflow/Dockerfile

```diff
@@ -21,15 +21,22 @@ RUN apt-get update && apt-get install -y --no-install-recommends git
 
 RUN pip3 install cloud-tpu-client pyyaml
 
+ARG tensorflow_text_version=latest
+
 # HACKs: Remove this once they are added to the Model Garden requirements
 RUN pip3 install tensorflow-recommenders --no-deps
-RUN pip3 install tensorflow-text==2.6.0rc0 --no-deps
+RUN if [ "${image_version}" = "nightly" -o "${tensorflow_text_version}" = "latest" ]; then \
+    pip3 install tensorflow-text --no-deps; \
+    else \
+    pip3 install tensorflow-text==${tensorflow_text_version} --no-deps; \
+    fi
 
+ 
 # Checkout tensorflow/models at HEAD
 WORKDIR /
 RUN curl -L https://github.com/tensorflow/models/archive/${model_garden_branch}.tar.gz | tar zx
 RUN mv models-${model_garden_branch} garden/
-RUN pip3 install -r /garden/official/requirements.txt
+RUN pip3 install -r /garden/official/requirements.txt --no-deps
```

Comment:
Have you tested this build with the `--no-deps` flag?

Reply:
Yes. That is something I was testing in the dashboard. It seems like --no-deps is not working out for us. 
Next option is to fix requirements.txt with proper version of the package which is indirectly installing 2.6.0.

Reply:
I have fixed the requirements.txt file in the model garden repo with commit: https://github.com/tensorflow/models/commit/31fd5b760458add99709c0b646381224355c008f#diff-6896191a14f6c66534bac457f50996b9330cd702cb6dbaae4c08d1d213e93d98